scripts/init.d/cgconfig.in

```diff
@@ -30,37 +30,54 @@
 ### END INIT INFO
 
 # get correct location of binaries from configure
-prefix=@prefix@;exec_prefix=@exec_prefix@;sbindir=@sbindir@
+sbindir=@sbindir@
 CGCONFIGPARSER_BIN=$sbindir/cgconfigparser
 CONFIG_FILE=/etc/cgconfig.conf
+CONFIG_DIR=/etc/cgconfig.d
 servicename=cgconfig
-lockfile=/var/lock/subsys/$servicename
 
+
+lockfile=/run/subsys/$servicename
 #
 # Source LSB routines
 #
-. /lib/lsb/init-functions
+SYSLIBFILE=/lib/lsb/init-functions
+OLDSYSLIBFILE=/etc/init.d/functions
+if [[ -x  $SYSLIBFILE ]] ; then
+  source $SYSLIBFILE
+elif  [[ -x $OLDSYSLIBFILE ]] ; then
+  source $OLDSYSLIBFILE
+  log_warning_msg() ( warning $@ ; printf "\n" 1>&2 ; )
+  log_failure_msg() ( failure $@ ; printf "\n" 1>&2 ; )
+  log_success_msg() ( success $@ ; printf "\n" 1>&2 ; )
```

Comment:
In the above three lines, the `$@` should be enclosed within double quotations.

Reply:
Done.